vmslave-config/ansible/group_vars/deb

```diff
@@ -1,2 +1,2 @@
 # on-imagebuilder is not working in ansible more than 2.0
-slave_ansible_version: 2.0.2
\ No newline at end of file
+ansible_version: 2.0.2
```

Comment:
deb build slave need ansible = 2.0.*, otherwise on-imagebuilder deb builf will fail.

but for other slaves we want to use newer(2.3.0) ansible for latest features.